{{cookiecutter.project_shortname}}/templates/invenio_pages/default.html

```diff
@@ -0,0 +1,33 @@
+{#
```

Comment:
I think we can just remove that and if someone wants to override it, they can define it here, wdyt?